src/app/containers/StakePage/index.tsx

```diff
@@ -553,7 +553,8 @@ interface FeeProps {
 function FeeBlock({ contractToken, usdTotal }: FeeProps) {
   const account = useAccount();
   const { t } = useTranslation();
-  const token = (contractToken.asset + '_token') as any;
+  const token = (contractToken.asset +
+    (contractToken.asset === Asset.SOV ? '_token' : '_itoken')) as any;
   const dollars = useCachedAssetPrice(contractToken.asset, Asset.USDT);
   const tokenAddress = getContract(token).address;
```

Comment:
```suggestion
  const tokenAddress = getContract(token)?.address;
```

Add error check to avoid page crashing